pkg/daemon/ceph/multus/utils.go

```diff
@@ -0,0 +1,286 @@
+package multus
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/containernetworking/plugins/pkg/ns"
+	"github.com/coreos/pkg/capnslog"
+	"github.com/vishvananda/netlink"
+	"gopkg.in/yaml.v2"
+	corev1 "k8s.io/api/core/v1"
+)
+
+var (
+	logger = capnslog.NewPackageLogger("github.com/rook/rook", "cephmultus")
+)
+
+const (
+	ifBase         = "mlink"
+	multusLinkName = "net1"
```

Comment:
is this always true?

Reply:
I have always found this to be the case, but have nothing the guarantee it. I will rewrite it to be more robust.